src/main/resources/application.yml

```diff
@@ -1,18 +1,28 @@
 server:
   port: 8080
   contextPath: /api

```

Comment:
I've seen those in PRs too. I'm not really sure what they are. They don't appear to be affecting functionality, though